packages/ui/src/working-groups/modals/ApplicationDetailsModal/ApplicationDetailsModal.tsx

```diff
@@ -21,9 +24,16 @@ type Tab = 'GENERAL' | 'FORM'
 export const ApplicationDetailsModal = React.memo(() => {
   const {
     hideModal,
-    modalData: { application },
+    modalData: { applicationId },
   } = useModal<ApplicationDetailsModalCall>()
 
+  const { loading, data } = useGetWorkingGroupApplicationQuery({
+    variables: { where: { id: applicationId } },
+  })
+  const application = data?.workingGroupApplicationByUniqueInput
+    ? asApplication(data.workingGroupApplicationByUniqueInput)
+    : null
```

Comment:
I guess this could be extracted and wrapped in a separate hook, further down the road

Reply:
Yeah, I'll write this down for the upcoming openings fix. Thanks!